docs/Components-Header.md

```diff
@@ -61,34 +158,85 @@ The `onLeftPress` is an optional prop that is used to set the function to be cal
 
 ```tsx
-<Header rightIcon="bullet" onRightPress={() => Alert.alert("pressed")} />
+<Header rightIcon="back" onRightPress={() => navigation.goBack()} />
 ```
 
-### `style`
+### `rightIconColor`
 
-The `style` prop is an optional prop that is used to set the style of the header container. This is a [`StyleProp<ViewStyle>`](https://reactnative.dev/docs/view-style-props) object.
+The `rightIconColor` is an optional prop that is used to set the tint color of the right navigation icon.
 
 ```tsx
 <Header
-  headerText="Header Title"
-  onLeftPress={() => navigation.goBack()}
-  style={{ backgroundColor: "purple" }}
+  rightIcon="back"
+  onRightPress={() => navigation.goBack()}
+  rightIconColor="white"
 />
 ```
 
-### `titleStyle`
+### `rightText`
 
-The `titleStyle` prop is an optional prop that is used to set the style of the header title. This is a [`StyleProp<TextStyle>`](https://reactnative.dev/docs/text-style-props) object.
+The `rightText` is an optional prop that is used to set the text for the right navigation button. Overrides the `rightIcon` prop.
+
+```tsx
+<Header rightText="Back" onRightPress={() => navigation.goBack()} />
+```
+
+### `rightTx`
+
+The `rightTx` is an optional prop that is used to lookup the translation for the right navigation button. Overrides the `rightIcon` and `rightText` prop`.
+
+```tsx
+<Header rightTx="header.back" onRightPress={() => navigation.goBack()} />
+```
+
+### `rightTxOptions`
+
+The `rightTxOptions` is an optional prop that is used to pass props to the translation lookup for the right navigation button. This is useful if you need to pass in dynamic values to the translation.
 
 ```tsx
 <Header
-  headerText="Header Title"
-  onLeftPress={() => navigation.goBack()}
-  titleStyle={{ color: "white" }}
+  rightTx="header.back"
+  rightTxOptions={{ name: "John" }}
+  onRightPress={() => navigation.goBack()}
+/>
+```
+
+### `RightActionComponent`
+
+The `RightActionComponent` is an optional prop that is used to set a custom component for the right navigation button. Overrides the `rightIcon`, `rightText`,  and `rightTx` props.
+
+```tsx
+<Header
+  RightActionComponent={() => <Text>Back</Text>}
+  onRightPress={() => navigation.goBack()}
 />
 ```
+
+### `onRightPress`
+
+The `onRightPress` is an optional prop that is used to set the function to be called when the right navigation button is pressed.
+
+```tsx
+<Header rightIcon="back" onRightPress={() => navigation.goBack()} />
+```
+
+### `safeAreaEdges`
+
+The `safeAreaEdges` optional prop can be used to override the default safe area edges. By default, the header will use the `top` safe area edge. If you want to use the `bottom` safe area edge, you can pass in `["bottom"]` to the `safeAreaEdges` prop.
```

Comment:
I'd love more info here on when you'd want to use the safeAreaEdges. (Generally -- I think it's when you have things like the notch on the top, or that weird bar on the bottom, but not if the view doesn't land on the top or bottom.)

Reply:
Yea, for the most part, Header will always be at the top; hence, the default. I don't know many usecases when it might need diff values. Maybe if there's a situation where you want to have a banner on top of the header - then you'd remove the safearea by doing `safeAreaEdges={[]}`.

Reply:
Good point.